Last-Update: 2013-09-14
Forwarded: not-needed
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: always install "doclib" module even if no data present
 in order to allow shipping help files in optional package.

--- a/plugins/wb.doclib/CMakeLists.txt
+++ b/plugins/wb.doclib/CMakeLists.txt
@@ -6,11 +6,11 @@
 set(SHARED_FILES
     res/DocLibrary/mysqldoclib.sqlite
     res/DocLibrary/mysqldoclib_webui.sqlite
 )
-if (EXISTS ${CMAKE_SOURCE_DIR}/plugins/wb.doclib/res/DocLibrary/mysqldoclib.sqlite)
-MESSAGE("Install DocLib")
 install(FILES ${PY_FILES} DESTINATION ${WB_PYTHON_MODULES_DIR})
+if (EXISTS ${CMAKE_SOURCE_DIR}/plugins/wb.doclib/res/DocLibrary/mysqldoclib.sqlite)
+MESSAGE("Install DocLib data")
 install(FILES ${SHARED_FILES} DESTINATION ${WB_PACKAGE_SHARED_DIR}/modules/data/DocLibrary)
 else()
-MESSAGE("Skipping installation of DocLib")
+MESSAGE("Skipping installation of DocLib data")
 endif()
